### PR TITLE
Fix for ED config 

### PR DESCRIPTION
**Pull request by para2x**
## Description

This PR is going to fix the css/pss path problem in settings for ED run. There is also another fix in settings package and specifically `site_pft_link_settings.R`.
## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [ ] When possible
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by para2x** [comment]
@serbinsh see if this solves the drop problem you had within the prepare settings. `drop` was function suggested by Alexy but not exported from purrr.

**Comment by serbinsh** [comment]
Great, will do @para2x

File: base/settings/R/site_pft_link_settings.R [hunk 1]
```diff
@@ -34,7 +34,7 @@ site.pft.link.settings <- function(settings) {
        unique()
      
      new.pfts <-  pft.l %>%
-       purrr::drop(~.x %in% def.pfts) %>%
+       purrr::discard(~.x %in% def.pfts) %>%
```
Comment (serbinsh) [inline base/settings/R/site_pft_link_settings.R]: @bailsofhay can you make this change in your version of this file in pecan then re-install pecan settings then trying running the LAI SDA where you actually run the prepare settings step and see if that fixes your current bug?

Comment (serbinsh) [reply base/settings/R/site_pft_link_settings.R]: This change fixed our issue with purrr in prepare.settings

**Pull request merged**